ci/run-tests.sh

```diff
@@ -5,6 +5,8 @@ cargo test \
     --features wasi-http \
     --features component-model \
     --features serve \
+    --features wasi-nn-onnx \
+    --features wasi-nn-winml \
```

Comment:
```suggestion
    --features wasmtime-wasi-nn/onnx \
```

See my previous comment about workspace features, plus I don't think we really want to enable the `wasmtime-wasi-nn/winml` feature unconditionally since (1) it is Windows-specific and (2) @jianjunz found that the WinML APIs aren't even available in the GitHub CI images.